lua/plugins/onmessage.lua

```diff
@@ -67,6 +67,18 @@ local function is_whitelisted(chat_id, text)
 	end
 end
 
+local function is_blacklisted(chat_id, text)
```

Comment:
Could've generalized the whitelist function and share some code between the two.